skill.xml

```diff
@@ -821,9 +821,9 @@
+			<Param Out="" Sid="SID_虚空" Name="MSID_JobSkill_Lindwurm" Help="MSID_H_JobSkill_Lindwurm" CommandName="MSID_JobSkill_Lindwurm" CommandHelp="MSID_H_JobSkill_Lindwurm" CommandWarning="" RootCommandSid="" IconKind="0" IconLabel="虚空" IconBmap="" Priority="0" Layer="0" Order="0" Cycle="0" Life="0" Timing="8" Target="0" Frequency="0" Stand="0" Action="1" Condition="武器の種類 == 魔道書 &amp;&amp; スキル確率( 80 )" ActNames="威力;" ActOperations="+;" ActValues="相手の魔防 * 0.5;" AroundCenter="0" AroundTarget="0" AroundCondition="" AroundName="" AroundOperation="" AroundValue="" GiveTarget="0" GiveCondition="" GiveSids="" RemoveSids="" SyncConditions="" SyncSids="" RebirthSid="" EngageSid="" ChangeSids="" CooperationSkill="" HorseSkill="" CovertSkill="" HeavySkill="" FlySkill="" MagicSkill="" PranaSkill="" DragonSkill="" AttackRange="" OverlapRange="" OverlapTerrain="" ZocRange="" ZocType="0" Work="0" WorkOperation="" WorkValue="0" Power="0" Rewarp="0" Removable="0" VisionCount="0" Cost="0" MoveSelf="0" MoveTarget="0" RangeTarget="0" RangeI="0" RangeO="0" RangeAdd="0" RangeExtend="0" Flag="0" Efficacy="0" EfficacyValue="0" EfficacyIgnore="0" BadState="0" BadIgnore="0" WeaponProhibit="0" EnhanceLevel="0" EnhanceValue.Hp="0" EnhanceValue.Str="0" EnhanceValue.Tech="0" EnhanceValue.Quick="0" EnhanceValue.Luck="0" EnhanceValue.Def="0" EnhanceValue.Magic="0" EnhanceValue.Mdef="0" EnhanceValue.Phys="0" EnhanceValue.Move="0" WeaponLevel.None="0" WeaponLevel.Sword="0" WeaponLevel.Lance="0" WeaponLevel.Axe="0" WeaponLevel.Bow="0" WeaponLevel.Dagger="0" WeaponLevel.Magic="0" WeaponLevel.Rod="0" WeaponLevel.Fist="0" WeaponLevel.Special="0" EquipIids="" Effect="汎用スキル" InheritanceCost="0" InheritanceSort="0" />
```

Comment:
Changed Lindwurm Class Skill:
    - Changed probability from `%DEX` to fixed `80%`
    - Scaled with `ENEMY RES` instead of `ENEMY MAG`